### ISSUE TITLE
Error importing @cfworker/json-schema when not using a bundler or transpiler

### ISSUE THREAD

**Issue by sgravrock** [cfworker/cfworker#77]
Steps to reproduce:

1. Install `@cfworker/json-schema`.
2. Create `test.mjs` as follows (or paste in any of the examples from the README):
```
import '@cfworker/json-schema';
```
3. Run `node test.mjs` (or `node --experimental-modules test.mjs` if using a version before 12).

I get the following error:
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '[...]/node_modules/@cfworker/json-schema/dist/deep-compare-strict' imported from [...]/node_modules/@cfworker/json-schema/dist/index.js
    at finalizeResolution (internal/modules/esm/resolve.js:276:11)
    at moduleResolve (internal/modules/esm/resolve.js:673:10)
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:784:11)
    at Loader.resolve (internal/modules/esm/loader.js:85:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:229:28)
    at ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:51:40)
    at link (internal/modules/esm/module_job.js:50:36) {
  code: 'ERR_MODULE_NOT_FOUND'
```

It looks like the TypeScript compiler is generating output that isn't compatible with Node's ES module loader. Specifically, ES module imports need to include the file extension. If I manually add the extension to each `import` or `export ... from` statement, e.g. changing `export * from './deep-compare-strict';` to `export * from './deep-compare-strict.js';`, the import succeeds.

**Comment by jdanyow** [comment]
fixed in 1.6.0
https://www.npmjs.com/package/@cfworker/json-schema/v/1.6.0

thanks for the report

**Comment by sgravrock** [comment]
Thanks for the quick fix!
